tigeros-repos-1.0/tigeros.repo

```diff
@@ -17,4 +17,3 @@ metadata_expire=7d
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-TigerOS-$releasever-primary
```

Comment:
Does this mean that the files need to be named RPM-GPG-KEY-TigerOS-3.0-primary?

Reply:
@ct-martin nope. *Usually* it would, but the `$releasever` variable refers to the Fedora version in this case.

Reply:
Actually, I will look more into this tonight.

It might need to be `RPM-GPG-Key-TigerOS-$vers-primary`